src/BlazorWebFormsComponents/ListView.razor

```diff
@@ -30,31 +21,47 @@ else if (Items == null || !Items.Any())
 else
 {
 
-	if (OnDataBinding.HasDelegate) OnDataBinding.InvokeAsync(EventArgs.Empty);
-
-	@foreach (var item in Items)
+	@if (LayoutTemplate is null)
 	{
+		LayoutTemplate = context => builder => builder.AddContent(1, context);
```

Comment:
This set a default LayoutTemplate if none was supplied - the default template being to add the `content`